.github/workflows/cicd-main.yml

```diff
@@ -649,6 +649,52 @@ jobs:
         rm -rf preds.json
       IS_OPTIONAL: true
 
+  # L2: Longform without TimeStamps from Audio Dir
+  L2_Longform_Speech_Transcription_Canary_Chunked_Infer_from_Audio_Dir:
+    needs: [pre-flight, cicd-test-container-build]
+    uses: ./.github/workflows/_test_template.yml
+    if: contains(fromJSON(needs.pre-flight.outputs.test_to_run), 'L2_Longform_Speech_Transcription_Canary_Chunked_Infer_from_Audio_Dir')
+    with:
+      RUNNER: self-hosted-azure
+      SCRIPT: |
+        python examples/asr/asr_chunked_inference/aed/speech_to_text_aed_chunked_infer.py \
+        pretrained_name=nvidia/canary-1b-flash \
```

Comment:
copy flash model to ci-cd testdata folder and avoid downloading each time.

Reply:
Mandatory~

Reply:
done.